### PR TITLE
#259 Implemented and tested Github Org/Orgs.

### PR DESCRIPTION
**Pull request by criske**
- Left puzzle to implement and test Github Org repos.

PR for #259

### CODE REVIEW

File: self-core-impl/src/main/java/com/selfxdsd/core/GithubOrganization.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+import javax.json.JsonObject;
+
+/**
+ * A Github Provider Organization.
+ *
+ * @author criske
+ * @version $Id$
+ * @since 0.0.9
+ * @todo #122:30 Implement and test GithubOrganization Repos.
```
Comment (amihaiemil) [inline self-core-impl/src/main/java/com/selfxdsd/core/GithubOrganization.java:14]: @criske Can you specify more details in this todo? 0pdd will throw an exception if the todo is too short. Usually, it has to have 10-15 words.

File: self-core-impl/src/main/java/com/selfxdsd/core/GithubOrganization.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ */
+public final class GithubOrganization implements Organization {
```
Comment (amihaiemil) [inline self-core-impl/src/main/java/com/selfxdsd/core/GithubOrganization.java:16]: @criske Make this class package-protected pls. It should not be visible outside of the core.

File: self-core-impl/src/main/java/com/selfxdsd/core/GithubOrganizations.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import java.util.Iterator;
+
+/**
+ * A Github Provider Organizations.
+ *
+ * @author criske
+ * @version $Id$
+ * @since 0.0.9
+ */
+public final class GithubOrganizations implements Organizations {
```
Comment (amihaiemil) [inline self-core-impl/src/main/java/com/selfxdsd/core/GithubOrganizations.java:19]: @criske This class also package-protected pls.

**Comment by amihaiemil** [comment]
@rultor merge it please

**Comment by criske** [comment]
Weird, initially the checkstyle passed on my end.

**Comment by amihaiemil** [comment]
@rultor merge

**Pull request merged**